glib/src/gobject/signal_group.rs

```diff
@@ -0,0 +1,185 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::signal::connect_raw;
+use crate::signal::SignalHandlerId;
+use crate::translate::*;
+use crate::Object;
+use crate::ObjectType;
+use crate::RustClosure;
+use crate::SignalGroup;
+use std::mem::transmute;
+
+impl SignalGroup {
+    #[cfg(any(feature = "v2_74", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v2_74")))]
+    #[doc(alias = "g_signal_group_connect_closure")]
+    pub fn connect(&self, signal_name: &str, after: bool, closure: RustClosure) {
+        unsafe {
+            gobject_ffi::g_signal_group_connect_closure(
+                self.to_glib_none().0,
+                signal_name.to_glib_none().0,
+                closure.as_ref().to_glib_none().0,
```

Comment:
Is there anything that has to be done here to ensure that the parameter/return types of the closure are matching with the signal?

Reply:
The type checking is handled by `RustClosure`, but I can't write a unit test for it because catching a panic from within the signal handler is UB and corrupts the internal gobject state

Reply:
Ok!

Reply:
Filed another issue for that here, btw: https://github.com/gtk-rs/gtk-rs-core/issues/645